tunnel/wstuncli.go

```diff
@@ -155,6 +160,33 @@ func NewWSTunnelClient(args []string) *WSTunnelClient {
 		wstunCli.Proxy = proxyURL
 	}
 
+	if *cliport != "" {
+		portList := strings.Split(*cliport, ",")
+		var ClientPorts []string
+		for _, v := range portList {
+			if strings.Contains(v, "..") {
+				k := strings.Split(v, "..")
+				bInt, err := strconv.Atoi(k[0])
+				if err != nil {
+					log15.Crit(fmt.Sprintf("Invalid Port Assignment: %q %v", *cliport, err))
```

Comment:
When we return error messages, it might be good to print the syntax. (i.e) `<number>..<number>`

Reply:
"Invalid Port Assignment: "56000..blah" strconv.Atoi: parsing "blah": invalid syntax"